spec_creation/autofill_eval_spec.py

```diff
@@ -156,9 +156,32 @@ def get_route_from_relation(r):
     return get_coords_for_relation(r["relation_id"],
         r["start_node"], r["end_node"])
 
-def validate_and_fill_leg(orig_leg):
-    # print(t)
+def _add_temporal_ground_truth(orig_loc, start_fmt_date, end_fmt_date):
+    # fill in timespan for which ground truth is valid (see issue #11)
+    # first, if loc is a dict, we need to convert end_loc to a list of dicts.
+    # otherwise, expect a list of dicts for each.
+
+    loc = copy.copy(orig_loc)
+
+    if isinstance(loc, dict):
+        loc = [loc]
+
+    print(loc)
+
+    # next, add dates if they do not exist
+    for l in loc:
+        if not l["properties"].get("valid_start_fmt_date"):
+            l["properties"]["valid_start_fmt_date"] = start_fmt_date
```

Comment:
We haven't finished discussing the design completely, but I think that the filled spec should store timestamps instead of a formatted string.

This related to the final step in the issue:
> Change the code that lookup up the ground truth for the evaluation (in emeval/input/spec_details.py - e.g. get_ground_truth_for_leg)

Recall that we will need to retrieve the ground truth in `get_ground_truth_leg` to evaluate the accuracy. It will be much easier to retrieve the correct ground truth object if we can compare timestamps directly.